packages/tests/wtr-config.js

```diff
@@ -1,57 +0,0 @@
-/**
```

Comment:
This file is referenced in a bunch of Wireit dependencies, can all be removed now.